.gitignore

```diff
@@ -1 +1,2 @@
 /build/
+settings.json
```

Comment:
Not sure I understand what that file is?

Reply:
This is vscode settings file. The goal is to ignore those files for vscode users.